application/backend/app/schemas/label.py

```diff
@@ -112,3 +81,26 @@ class LabelReference(BaseRequiredIDModel):
             }
         }
     }
+
+
+class PatchLabels(BaseModel):
+    labels_to_edit: list[LabelToEdit] = Field(default_factory=list, description="List of labels to edit")
+    labels_to_add: list[LabelCreate] = Field(default_factory=list, description="List of labels to add")
+    labels_to_remove: list[LabelToRemove] = Field(default_factory=list, description="List of labels to remove")
```

Comment:
Shall we follow the same naming convention for request schemas and remove `To` suffix, e.g.: `LabelEdit` instead of `LabelToEdit`?